manifests/mount.pp

```diff
@@ -31,6 +31,7 @@
       exec {"create ${mountpoint} and parents":
         command => "mkdir -p ${mountpoint}",
         unless  => "test -d ${mountpoint}",
+        path    => ['/usr/bin','/bin'],
```

Comment:
It doesn't work for RHEL 7 if this is not there. The path for mkdir  is different (it is now under /usr/bin/mkdir). That's why I want this in this patch. I am happy to change for $::path though.